public/docs/ts/latest/guide/router.jade

```diff
@@ -1041,61 +1241,50 @@ a#child-routing-component
-
-  There's a practical benefit to restricting its scope in this way.
-
-  First we can evolve the service independently of the rest of the application
-  without fear of breaking what should be unrelated modules.
-
-  Second, we can delay loading this service into memory until we need it.
-  We can remove it from the application launch bundle,
-  reducing the size of the initial payload and improving performance.
-  We can load it optionally, [asynchronously](#asynchronous-routing) with the other *Crisis Center* components
-  if and when the user begins that workflow.
-
-:marked
   ### Child Route Configuration
 
   The `CrisisCenterComponent` is a *Routing Component* like the `AppComponent`.
   It has its own `RouterOutlet` and its own child routes.
 
+  Add the following `crisis-center-home.component.ts` to the `crisis-center` folder.
+
++makeExcerpt('app/crisis-center/crisis-center-home.component.ts (minus imports)', 'minus-imports')
+
+:marked
   We create a `crisis-center.routing.ts` file as we did the `heroes.routing.ts` file.
   But this time we define **child routes** *within* the parent `crisis-center` route.
 
 +makeExcerpt('app/crisis-center/crisis-center.routing.1.ts (Routes)', 'routes')
 
 :marked
   Notice that the parent `crisis-center` route has a `children` property
-  with an array of two routes.
+  with a single route containing our `CrisisListComponent`. The `CrisisListComponent` route
+  also has a `children` array with two routes.
+
   These two routes navigate to the two *Crisis Center* child components,
-  `CrisisListComponent` and `CrisisDetailComponent`.
+  `CrisisCenterHomeComponent` and `CrisisDetailComponent`.
 
   There are some *important differences* in the treatment of these routes.
 
-  First, the router displays the components of these child routes in the `RouterOutlet`
+  The router displays the components of these routes in the `RouterOutlet`
   of the `CrisisCenterComponent`, not in the `RouterOutlet` of the `AppComponent` shell.
 
-  Second, the child paths *extend* the path of their parent route.
+  The `CrisisListComponent` contains our crisis list and `RouterOutlet` to
+  display our `Crisis Center Home` and `Crisis Detail` routes.
+
+  The `Crisis Detail` route is a child of the `Crisis List`. Since the router [reuses components](#reuse)
+  by default, the `Crisis Detail` component will be re-used as we select different crises. In
+  our `Hero Detail` route, the component was recreated each time we selected a different hero.
+
+  The child paths *extend* the path of their parent route unless the are _empty_.
 
   Normally paths that begin with `/` refer to the root of the application.
   Here they are appended to the path to the `CrisisCenterComponent`.
 
-  To write an URL that navigates to the `CrisisListComponent`, we'd append its child route path,  `/`,
+  To write an URL that navigates to the `CrisisCenterHomeComponent`, we'd append its child route path, `/`,
```

Comment:
I found myself re-reading this slash section a few times to follow it. I don't have any good ideas on how to clarify it though :(

Just pointing it out as this was the first part of the chapter I had to re-read.

Loving this chapter!